### ISSUE TITLE
Cannot create a list

### ISSUE THREAD

**Issue by charignon** [kiblee/tod0#38]
**Describe the bug**

I cannot create a list, everything else seems to work:
```
$ todocli newl "foo"
Traceback (most recent call last):
  File "/opt/homebrew/opt/python@3.10/Frameworks/Python.framework/Versions/3.10/bin/todocli", line 8, in <module>
    sys.exit(main())
  File "/opt/homebrew/lib/python3.10/site-packages/todocli/cli.py", line 267, in main
    namespace.func(namespace)
  File "/opt/homebrew/lib/python3.10/site-packages/todocli/cli.py", line 138, in newl
    todo_api.create_list(args.list_name)
  File "/opt/homebrew/lib/python3.10/site-packages/todocli/todo_api.py", line 137, in create_list
    return request.execute()
  File "/opt/homebrew/lib/python3.10/site-packages/todocli/rest_request.py", line 49, in execute
    return self.evaluateResult(o)
  File "/opt/homebrew/lib/python3.10/site-packages/todocli/rest_request.py", line 15, in evaluateResult
    o.raise_for_status()
  File "/opt/homebrew/lib/python3.10/site-packages/requests/models.py", line 953, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://graph.microsoft.com/v1.0/me/todo/lists
```

**Comment by kiblee** [comment]
I can't reproduce this error. Is this still happening?

**Comment by charignon** [comment]
I can still reproduce the error

**Comment by kiblee** [comment]
This might have been fixed with #37. Could you update `tod0` and try again?
